core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -1800,6 +1808,32 @@ public Future<Void> asyncRead(RemoteStorageFetchInfo fetchInfo, Consumer<RemoteL
                 new RemoteLogReader(fetchInfo, this, callback, brokerTopicStats, rlmFetchQuotaManager, remoteReadTimer));
     }
 
+    void validateCopyAndExpireTasks(TopicIdPartition topicPartition, boolean tieredStateOfTopic) {
```

Comment:
can we rename the `topicPartition` to `topicIdPartition` to avoid confusion?

Reply:
Yea we can, but I see this is the same pattern applied in other methods doHandleLeaderPartition doHandleFollowerPartition. I can update there too. Any suggestion on the name ?
for ex here : 
followerRLMTasks.computeIfAbsent(topicPartition, topicIdPartition -> {